config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
+  resources :recipes
```

Comment:
Since we are not going to implement `update` action, consider limiting the actions for this resource.